scripting.md

```diff
@@ -0,0 +1,104 @@
+# SuperTux Level Editor Scripting Tutorial
+
+## What is Scripting?
+
+Scripting in SuperTux is when a game object is altered using things like a script trigger or a switch.  Scripting is written in the Squirrel coding language.
+
+#### What sort of things use scripting?
+
+Scripts can be activated by the following things, plus many more:
+
+* Script Trigger, which activates the script.
+* Switch/Button, which activate scripts when used.
+* Death Scripts, which activate when an enemy dies.
+* Init-Scripts, which activate when Tux enters the sector.
+
+In order to edit a script, right click a scriptable object and open the script editor.
+
+## Structure of scripts
+
+The basic structure of a line of script is as so:
+
+    object.action(Number);
+
+"object" is what you are going to activate, "action" is what the object does, and "Number" is the property of the action, like a number, series of numbers, or words. The semicolon activates the script, you have to use it in most situations.
+
+### Some scriptable objects
+
+Here are some things you can script in a sector:
+
+* Light: 
+
+      settings.set_ambient_light(color1, color2, color3);
```

Comment:
You're starting with two examples that does **not** match the scheme you described above, that is going to be confusing for beginners.

Reply:
2 out of 3 of them do. i added an explanation.